pysat/tests/classes/cls_instrument_library.py

```diff
@@ -512,6 +513,72 @@ def test_clean_warn(self, clean_level, inst_dict, caplog):
 
         return
 
+    @pytest.mark.second
+    # Need to maintain download mark for backwards compatibility.
+    # Can remove once pysat 3.1.0 is released and libraries are updated.
+    @pytest.mark.load_options
+    @pytest.mark.download
+    @pytest.mark.parametrize('pad', [{'days': 1}, dt.timedelta(days=1)])
+    def test_load_w_pad(self, pad, inst_dict):
+        """Test that instruments load at each cleaning level.
+
+        Parameters
+        ----------
+        pad : pds.DateOffset, dict, or NoneType
+            Valid pad value for initializing an instrument
+        inst_dict : dict
+            Dictionary containing info to instantiate a specific instrument.
+            Set automatically from instruments['download'] when
+            `initialize_test_package` is run.
+
+        """
+        # Skip for Python 3.6, keeping information that will allow adding
+        # or skipping particular instruments.
+        # TODO(#1136): Remove skip once memory management is improved
```

Comment:
Fixing this test in Python 3.6 will require a major overhaul in pysat, and so cannot be tackled at the moment.  Recorded here: https://github.com/pysat/pysat/issues/1136